tests/data/charm.yaml

```diff
@@ -13,8 +13,6 @@ applications:
   kubernetes-control-plane:
     options:
       enable-metrics: false
-      controller-manager-extra-args: |-
-        requestheader-client-ca-file=/root/cdk/ca.crt requestheader-allowed-names=system:kube-controller-manager,client
   ################################################
```

Comment:
these lines become unnecessary after this merge:
* https://github.com/charmed-kubernetes/charm-kubernetes-control-plane/pull/257